File: card/src/main/java/io/github/susimsek/card/CardApplication.java
package io.github.susimsek.card;

import io.github.susimsek.card.dto.CardContactInfoDTO;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

@EnableDiscoveryClient
@EnableConfigurationProperties(value = CardContactInfoDTO.class)
@SpringBootApplication
public class CardApplication {

	public static void main(String[] args) {
		SpringApplication.run(CardApplication.class, args);
	}

}

File: card/src/main/kotlin/io/github/susimsek/card/CardApplication.kt
package io.github.susimsek.card

import io.github.susimsek.card.dto.CardContactInfoDTO
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@EnableDiscoveryClient
@EnableConfigurationProperties(value = [CardContactInfoDTO::class])
@SpringBootApplication
class CardApplication
fun main(args: Array<String>) {
    runApplication<CardApplication>(*args)
}
